.github/workflows/ci.yml

```
@@ -0,0 +1,187 @@
+name: Hedera Services CI
+
+on: [push]
+
+jobs:
+  Run-Unit-Tests:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_DB: fcfs
+          POSTGRES_PASSWORD: password
+          POSTGRES_USER: swirlds
+          PGDATA: /var/lib/postgresql/data/pgdata
+          POSTGRES_PORT: 5432
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'adopt'
+          cache: maven
+      - name: Build with Maven
+        run: mvn install
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v2
+        with:
+          root_dir: '/home/runner/work/hedera-services/'
+          files: 'jacoco.xml'
+      - name: Upload build
+        uses: actions/upload-artifact@v2
+        with:
+          name: build
+          path: |
+            **/target/*.jar
+            **/target/pom.xml
+  Javadoc:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'adopt'
+          cache: maven
+      - name: Generate javadocs
+        run: |
+          mvn clean install -DskipTests
+          mvn javadoc:javadoc -DskipTests
+  Run-E2E-Tests-1:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_DB: fcfs
+          POSTGRES_PASSWORD: password
+          POSTGRES_USER: swirlds
+          PGDATA: /var/lib/postgresql/data/pgdata
+          POSTGRES_PORT: 5432
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - uses: actions/checkout@v2 # checkout current branch
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'adopt'
+          cache: maven
+      - name: Build docker images # build the image for hedera-node
+        run: bash compose-build.sh
+      - name: Start network # start 3 node network
+        run: docker-compose -f docker-compose-ci.yml up -d
+      - name: Run E2E Suites # run selected e2e suites
+        run: |
+          mvn clean install -DskipTests
+          cd test-clients
+          mvn test -Dargs="-s contract.opcodes, contract.openzeppelin" -Pe2e-tests
```

Is it possible to combine?
```suggestion
          ./mvnw clean test -pl test-clients -Dargs="-s contract.opcodes, contract.openzeppelin" -Pe2e-tests
```

---

The first maven command has been removed. However, the `cd test-clients` command still needs to be present, since E2E tests need to be executed from the context of `test-clients`, because they access resources from `test-clients` in order to execute successfully.